### DISCUSSION TITLE
Endpoint Device

### DISCUSSION THREAD

**Discussion by alxndr-stlz** [jwallet/spy-spotify#404] [Q&A]
Hello,
when I try to record my Playlist, Spytify, only says that it can't record because Spotify doesn't use the same endpoint as Spytify.
I changed the Spytify Endpoint & the Endpoint of Spotify to a VR-Cable (works) but Spoytify still tells me that its not the same.
![Screenshot_2](https://user-images.githubusercontent.com/49165798/130961223-f6b51ca8-622a-40b3-837f-9558cd31ba4c.png)

![Screenshot_1](https://user-images.githubusercontent.com/49165798/130960952-508f4b37-2348-49c8-9605-fc5e4f2f4332.png)

**Comment by jwallet** [comment]
did you isolate Spotify Desktop App as well, like shown in this gif 
https://jwallet.github.io/spy-spotify/faq.html#isolate-spotify-audio-endpoint
